projects/pydateutil/fuzz_prop_isoparse.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import atheris
+
+from hypothesis import given, assume
+from hypothesis import strategies as st
+
+with atheris.instrument_imports():
+  from dateutil import tz
+  from dateutil.parser import isoparse
+
+
+# Strategies
+TIME_ZONE_STRATEGY = st.sampled_from([None, tz.UTC] +
+    [tz.gettz(zname) for zname in ('US/Eastern', 'US/Pacific',
```

Comment:
I think it is preferable to use `"America/New_York"` and `"America/Los_Angeles"` rather than the deprecated `US/...` aliases.